docs/products/kafka/howto/rotate-credentials.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Rotate credentials for an Apache Kafka subscription
+sidebar_label: Rotate credentials
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import RelatedPages from "@site/src/components/RelatedPages";
+
+Rotate credentials for an Apache Kafka subscription to replace outdated credentials and keep access secure and approved.
+The request must be approved by another member of the owner group before new
+credentials can be downloaded.
+
+## Prerequisites
+
+You must be a member of the **owner group** of the access to rotate.
+
+## Request a credential rotation
+
+1. In the [Aiven Console](https://console.aiven.io/), go to
+   **Tools > Apache Kafka governance operations**.
+1. In the sidebar, click <ConsoleLabel name="Streaming catalog"/> > **Access overview**.
+1. In the **Access overview** page, locate the service user for which you need
+   credentials.
+1. Click <ConsoleLabel name="actions"/> > **Request credential rotation**.
+1. In the **Request credential rotation** dialog, enter a message for the approvers.
+1. Click **Submit**.
+
+### What happens after you submit a rotation request
```

Comment:
There is just one `###` heading inside `## Request a credential rotation`. Consider changing `###` into a plain paragraph with the list items or a large note with the same.